src/library_gl.js

```diff
@@ -715,9 +715,22 @@ var LibraryGL = {
     switch(name_) {
       case 0x1F00 /* GL_VENDOR */:
       case 0x1F01 /* GL_RENDERER */:
-      case 0x1F02 /* GL_VERSION */:
         ret = allocate(intArrayFromString(GLctx.getParameter(name_)), 'i8', ALLOC_NORMAL);
         break;
+      case 0x1F02 /* GL_VERSION */:
+        var glVersion = GLctx.getParameter(GLctx.VERSION);
+        // return GLES version string corresponding to the version of the WebGL context
+#if USE_WEBGL2
+        if (GLctx.canvas.GLctxObject.version >= 2) {
```

Comment:
The browser version is reported in the vendor-specific field, so the information is not totally lost.
The problem is WebGL and GL ES have different version numbering, so it's difficult to make a proper ES version from the WebGL version if it's different than 1.0 or 2.0.
The situation is a little different with GLSL, we can probably just report the same version number as WebGL.